pkg/proxy/nftables/proxier.go

```diff
@@ -1422,6 +1533,55 @@ func (proxier *Proxier) syncProxyRules() {
 			}
 		}
 
+		if svcInfo.SessionAffinityType() == v1.ServiceAffinityClientIP {
+			// Generate the per-endpoint affinity sets
```

Comment:
You're not "generating" them here...

You could merge this with the earlier `allLocallyReachableEndpoints` loop (that does the `ensureChain`s for the endpoint chains), and then have the comment there be "Note the endpoints chains and affinity sets that will be used"